src/wallet/wallet.h

```diff
@@ -160,7 +160,7 @@ class JSOutPoint
     // Transaction hash
     uint256 hash;
     // Index into CTransaction.vjoinsplit
-    size_t js;
+    uint64_t js;
```

Comment:
I believe we have discussed this issue before; I agree that this should be done so serialization is unambiguous, but we should check that this does indeed match existing serialization.